tasks/section_08_level2.yml

```diff
@@ -235,6 +236,7 @@
   - name: 8.3.1 Install AIDE (init) (Scored)
     command: aideinit
     when: aide_installed.changed == True
+    when: use_aide == True
```

Comment:
Can you change this to a single `when:` clause with a list to keep the same template as lines 246-252?